src/data/shared/summit-2024-navigation.js

```diff
@@ -0,0 +1,8 @@
+export const navigation = [
+  { name: 'Information', href: '/summit-2023/#information' },
```

Comment:
The drop-down menu for Information takes us to last year's and I think this link is why